assembl/lib/frontend_urls.py

```diff
@@ -133,7 +133,7 @@ def __init__(self, discussion):
         'sentrytest': '/sentrytest',
         'groupSpec': '/G/*remainder',
         # 'purl_posts': '/posts*remainder',
-        'purl_idea': '/idea*remainder',
+        # 'purl_idea': '/idea*remainder',
```

Comment:
Why remove the route to an idea?

Reply:
Because the backbone routes take higher priority than react routes. I removed it so that we can handoff the purls to a view that will check the logic and redirect to V1 or V2 accordingly.